distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -13,6 +13,7 @@ import Standard.Table.Data.Expression.Expression
 import Standard.Table.Data.Expression.Expression_Error
 import Standard.Table.Data.Join_Condition.Join_Condition
 import Standard.Table.Data.Join_Kind.Join_Kind
+import Standard.Table.Data.Match_Columns as Match_Columns_Helpers
```

Comment:
Why aliased?

Reply:
Because we import `Match_Columns` type just a few lines above.
And we need to distinguish the type from the module.

I'd like to keep the helpers static on the module to keep them more hidden - if they are on the type they are easier to use by users which they are not intended for that.